astro/src/content/docs/extend/code/lambdas/self-service-registration.mdx

```diff
@@ -23,11 +23,11 @@ function validate(result, user, registration, formContext) {
 
 This lambda must contain a function named `validate` that takes four parameters. The parameters that the lambda is passed are:
 
```

Comment:
The section below doesn't follow the normal lambda doc format of:

* list of parameters including whether they are read-only or not
* prose describing the parameters, including links to doc

This is enough of an 'odd-duck' lambda that may be okay, but wanted to call it out.

Reply:
I think you mean, they are all sort of munged together in the list of parameters? We do document if they are read-only or not, but it does look like we have links to the data types so to speak inline instead of as a separate secion.

Reply:
I did a little work to make them closer.